include/cl/storage.hh

```diff
@@ -93,23 +93,19 @@ struct Var {
      *
      * @todo a better API for initializers?
      */
-    bool                        initialized;
+    bool                        initialized = false;
 
     /**
      * true if the variable is external (defined in another module)
      */
-    bool                        isExtern;
+    bool                        isExtern = false;
 
     /**
      * true if there is at least one instruction in the program that takes an
      * address of the variable (or some part of it)
      */
-    bool                        mayBePointed;
+    bool                        mayBePointed = false;
 
-    /**
-     * dummy constructor
-     * @note known to be useful for internal purposes only
-     */
     Var();
     ~Var();
```

Comment:
You can write it like this to be more in spirit of C++11 (and remove the empty definitions in the respective `.cc`)
```suggestion
    Var() = default;
    ~Var() = default;
```

Reply:
The initial idea was to make their implementation hidden, so that we do not need to update a public header file and rebuild all software that uses it, each time the implementation of `Var()` or `~Var()` changes.  But, given the fact that nothing except CL and Predator uses the header file these days, it is not so important any more.